Emby.Server.Implementations/Data/SqliteUserDataRepository.cs

```diff
@@ -33,14 +32,9 @@ public class SqliteUserDataRepository : BaseSqliteRepository, IUserDataRepositor
         /// Opens the connection to the database
         /// </summary>
         /// <returns>Task.</returns>
-        public void Initialize(ReaderWriterLockSlim writeLock, ManagedConnection managedConnection, IUserManager userManager)
```

Comment:
I believe the original approach taken here results in two 'write' connections being created to library.db.  Whilst this is safe in practice (we have mutex protection etc.) it may result in conflicts when both (SqliteItemRepo and SqliteUserDataRepo) want to write at the same time; one shall receive a LOCK error status.

All the other db files only have one connection being made to them.  Could we modify library to be the same?

Reply:
My comment above is more about the instantiation, creating two db connections:
`public class SqliteUserDataRepository : BaseSqliteRepository, IUserDataRepository`

Reply:
This is... dirty, 2 classes accessing the same DB and sharing a lock.

Reply:
We either need to raise a ticket so this issue is not forgotten, or add a ToDo (if anyone cares checking those)..